externs/olcsx.js

```diff
@@ -5,6 +5,45 @@
  */
 var olcsx;
 
+
+/**
+ * @typedef {{
+ *   map: (!ol.Map),
+ *   target: (Element|string|undefined),
+ *   createSynchronizers: (undefined|function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)
```

Comment:
You checked the only 2 cases where `undefined` is at the beginning in ol3.
On the contrary, putting `undefined` at the end is done 24 times.
Checked with `git grep -i 'undefined|.*function'` and `git grep -i '.*Function|undefined'`